webcrypto/subtle_crypto_test.go

```diff
@@ -92,3 +92,95 @@ func TestSubtleCryptoImportExport(t *testing.T) {
 		assert.NoError(t, gotScriptErr)
 	})
 }
+
+func TestSubtleCryptoEncryptDecrypt(t *testing.T) {
+	t.Parallel()
+
+	t.Run("AES CBC", func(t *testing.T) {
```

Comment:
nit: I think we can put in dedicated funcs instead of nesting them.

Reply:
The benefit I see in doing it this way is that the test file, which is gonna end up being very big once all the algorithms and functionalities are implemented, remains nicely segmented. Another benefit is that it avoids longer test function names such as `TestSubtleCryptoEncryptDecryptAESGCMiv96bits`. If you don't oppose it, I'd rather keep it this way, let me know 😸